css/owned-card.styl

```diff
@@ -64,6 +64,14 @@
   .name
     margin-bottom: 1em
     height: 2.9em
+    overflow: hidden
+    text-overflow: ellipsis
+    display: -webkit-box
+    -webkit-box-orient: vertical
```

Comment:
[nib](https://tj.github.io/nib/) will handle browser prefixing, so `box-orient` is what you want to use here.  `line-clamp` is pretty unsupported.   [A cross-browser solution](http://hackingui.com/front-end/a-pure-css-solution-for-multiline-text-truncation/) is pretty involved.